packages/utils/src/descriptors.ts

```diff
@@ -94,5 +94,5 @@ export function getConcordanceTheme() {
 }
 
 export function diffDescriptors(actual: unknown, expected: unknown, options: DisplayOptions): string {
-  return concordanceModule.diff(expected, actual, options)
+  return concordanceModule.diff(actual, expected, options)
```

Comment:
Refer to the following code, the first parameter should be actual, the second parameter should be expected

https://github.com/concordancejs/concordance/blob/791d2a89b40eb13f2c889ac270dd8be190cf8073/lib/diff.js#L388-L389

However, the test failed after the modification. According to the error message, it seems to be related to the content of `html.meta.json.gz`. I don't know how to deal with this?

Reply:
Run vitest with `-u` flag and update snapshots then

Reply:
Some files seem to be deleted after running, I'm not sure if this is correct.

Reply:
You should've run it only inside the test package that was failing, not all test packages,

Reply:
According to the error message, it seems that the two tests `mocked.test.ts` and `diff.test.ts` failed, but I ran these two tests locally and passed. I don’t know if there is a problem with my local environment. 😕

Reply:
Seems like you have some problems with your local env, check your branch, and rebuild the project.

Reply:
It seems to be ok after reinstalling dependencies and rebuilding.